src/ceph-daemon

```diff
@@ -131,15 +133,21 @@ def call(command, desc=None, verbose=False, **kwargs):
                 if not isinstance(message, str):
                     message = message.decode('utf-8')
                 if fd == process.stdout.fileno():
+                    message = out_buffer + message
                     out += message
```

Comment:
Need to append to `out` before adding `out_buffer` to message:

```suggestion
                     out += message
                     message = out_buffer + message
```

If out_buffer has been set, the same repeating substr pattern repeats in the returned `out`